lib/util/Components.js

```diff
@@ -696,7 +697,9 @@ function componentRule(rule, context) {
   const ruleInstructions = rule(context, components, utils);
   const updatedRuleInstructions = util._extend({}, ruleInstructions);
   const propTypesInstructions = propTypes(context, components, utils);
-  const allKeys = new Set(Object.keys(detectionInstructions).concat(Object.keys(propTypesInstructions)));
+  const defaultPropsInstructions = defaultProps(context, components, utils);
+  const allKeys = new Set(Object.keys(detectionInstructions).concat(Object.keys(propTypesInstructions))
+    .concat(Object.keys(defaultPropsInstructions)));
```

Comment:
this doesn't need to be two concat calls, it can be `arr1.concat(arr2, arr3)`

Reply:
Oh 🤦‍♂️